src/app/core/components/navigation/style.scss

```diff
@@ -30,8 +30,15 @@ header {
     display: block;
     height: 30px;
     margin-left: variables.$content-padding - 10px;
+    margin-right: variables.$content-padding - 10px;
+    padding-right: 160pt;
     width: 200px;
   }
+
+  h3 {
+    font-size: variables.$font-size-head -10pt;
+    font-weight: normal;
```

Comment:
Use div instead of h3 if font-weight needs to be normal?

Reply:
Use class name to reference div inside stylesheet, so when anything else will be added it will not be affected.